Sources/SwiftInspectorVisitors/FunctionDeclarationVisitor.swift

```diff
@@ -87,6 +91,7 @@ public struct FunctionDeclarationInfo: Codable, Hashable {
 
   public struct ArgumentInfo: Codable, Hashable {
     public let argumentLabelName: String
+    public let parameterName: String
```

Comment:
I was going to say that we should make the argument label optional but since Swift stresses both exist I think this is fine.
<img width="788" alt="Screen Shot 2022-09-16 at 2 37 30 PM" src="https://user-images.githubusercontent.com/1791049/190791220-1390e44a-cd6c-49ed-9e61-b2bc3647b833.png">

Reply:
Per my realization at the end of https://github.com/fdiaz/SwiftInspector/pull/136#discussion_r973431564 I think making the `parameterName` optional would be helpful.

Reply:
I just pushed a new commit that hopefully will help clarify a bit https://github.com/fdiaz/SwiftInspector/pull/136/commits/f4c285dd18f85cbd4fd352cb0026d2286ea6e77e